src/stepper/index.tsx

```diff
@@ -43,6 +51,7 @@ export default class Stepper extends PureComponent <StepperProps, StepperState>
 
   componentDidMount() {
     this.update(this.state.value, undefined, false)
+    document.addEventListener('keydown', this.handleKeyDown)
   }
 
   componentWillReceiveProps(nextProps: StepperProps) {
```

Comment:
btw, can you migrate this into a componentDidUpdate ? :)

Reply:
Done